src/fobnail-attester.c

```diff
@@ -225,6 +290,7 @@ int main(int UNUSED argc, char UNUSED *argv[])
     att_coap_add_resource(coap_context, COAP_REQUEST_FETCH, "attest", coap_attest_handler);
     att_coap_add_resource(coap_context, COAP_REQUEST_FETCH, "ek", coap_ek_handler);
     att_coap_add_resource(coap_context, COAP_REQUEST_FETCH, "aik", coap_aik_handler);
+    att_coap_add_resource(coap_context, COAP_REQUEST_FETCH, "meta", coap_meta_handler);
```

Comment:
I think URI is `metadata`, @arturkow2000 please confirm.

Reply:
@krystian-hebel  @arturkow2000  Fixed.